_posts/2018-02-23-user-meeting-registration.md

```diff
@@ -4,7 +4,7 @@ title: 2018 OME Annual Users Meeting registration
 intro-blurb: The OME team is pleased to announce that registration for the 2018 OME Annual Users Meeting is now open
 ---
 
-We are pleased to announce that registration for the 2018 OME Annual Users Meeting is now open. [Register now](https://www.buyat.dundee.ac.uk/conferences-and-events/school-of-life-sciences/ome-annual-users-meeting/2018-ome-annual-users-meeting).
+We are pleased to announce that registration for the 2018 OME Annual Users Meeting is now open.
```

Comment:
This caught me for a minute. Not sure if this shouldn't be "is now closed" but probably now overly important.

Reply:
Yes we are now managing registration states via temporary banners rather than posts. Happy to switch to `closed` to reduce confusion.

Reply:
See 8b854ff